frontend/src/containers/loading-spinner/LoadingSpinner.js

```diff
@@ -1,32 +1,32 @@
-import React, { Component } from 'react';
+import React from 'react';
 import styles from './_loading-spinner.scss';
 import { HashLoader } from 'react-spinners';
+import { usePromiseTracker } from "react-promise-tracker";
 import classNames from 'classnames/bind';
 
 let cx = classNames.bind(styles);
 
-class LoadingSpinner extends Component {
+const LoadingSpinner = props => {
 
-  render() {
+  const { promiseInProgress } = usePromiseTracker();
 
-    return (
-      <section className={styles['loading-spinner-root-container']}>
-        {this.props.displaySpinner && (
-          <div className={styles['spinner-container']}>
-            <div className={styles['spinner-container__content']}>
-              <HashLoader
-                color={'#0090c1'}
-              />
-              <span>Loading your data...</span>
-            </div>
+  return (
+    <section className={styles['loading-spinner-root-container']}>
+      {promiseInProgress && (
+        <div className={styles['spinner-container']}>
+          <div className={styles['spinner-container__content']}>
+            <HashLoader
+              color={'#0090c1'}
```

Comment:
Do we want to set the color code in a color file and name the variable so we don't have a magic number?

Reply:
Added it as a variable for more clarity.